webapp/components/wpt-interop.html

```diff
@@ -269,8 +289,11 @@
 
     async ready() {
       await super.ready();
+      this.loadingCount++;
+      const that = this;
```

Comment:
If you don't need the `arguments` object in `function(r) {...}` below, then prefer using `r => {...}` and dropping `that = this`.

Reply:
Done.